packages/app/src/modules/options/config.js

```diff
@@ -0,0 +1,49 @@
+import {
+    VIS_TYPE_COLUMN,
+    VIS_TYPE_BAR,
+    VIS_TYPE_STACKED_COLUMN,
+    VIS_TYPE_STACKED_BAR,
+    VIS_TYPE_PIVOT_TABLE,
+    VIS_TYPE_LINE,
+    VIS_TYPE_RADAR,
+    VIS_TYPE_AREA,
+    VIS_TYPE_PIE,
+    VIS_TYPE_GAUGE,
+    VIS_TYPE_YEAR_OVER_YEAR_LINE,
+    VIS_TYPE_YEAR_OVER_YEAR_COLUMN,
+    VIS_TYPE_SINGLE_VALUE,
+} from '@dhis2/analytics';
+
+import pivotTableConfig from './pivotTableConfig';
+import columnConfig from './columnConfig';
+import stackedColumnConfig from './stackedColumnConfig';
+import lineConfig from './lineConfig';
+import areaConfig from './areaConfig';
+import pieConfig from './pieConfig';
+
+export const getOptionsByType = type => {
```

Comment:
could this (along with all the imported files) be centralised in Analytics for reusability?

Reply:
Potentially yes. I think we need to discuss what to do with all the shared code that we are going to have as soon as we start rewriting Event Reports and Event Visualizer.
There is also the idea of having a visualizer app shell of some sort, with all the shared stuff, and apps that build on top of it. In that case could be that it makes more sense to have this more app related stuff in there...